app/controllers/api/v1/project_roles_controller.rb

```diff
@@ -10,9 +10,13 @@ def resource_name
   end
 
   def update
-    super do
-      UserAddedToProjectMailerWorker.perform_async(api_user.id, controlled_resource.resource.id, roles) if new_roles_present?(roles)
-    end
+    UserAddedToProjectMailerWorker.perform_async(acl_user_id, controlled_resource.resource.id, roles) if new_roles_present?(roles)
```

Comment:
Also i'm not sure i'm following the `acl_user_id`, don't we just want to send an email to the linked user in the params?